products/sentinel_one.py

```diff
@@ -434,6 +435,11 @@ def _process_queries(self):
                 # merge all query tags into a single string
                 merged_tag = Tag(','.join(tag.tag for tag in merged_tags), ','.join(str(tag.data) for tag in merged_tags))
 
+                if len(query_base):
```

Comment:
with the current setup of the nested_process_search function, it does not add the query_base to the function self._queries correctly. if adding it as a parameter in the existing append line it will add the query_base multiple times.